controllers/vrg_volrep.go

```diff
@@ -1913,19 +1922,13 @@ func (v *VRGInstance) restorePVsAndPVCsFromS3(result *ctrl.Result) (int, error)
 func (v *VRGInstance) restorePVsFromObjectStore(objectStore ObjectStorer, s3ProfileName string) (int, error) {
 	pvList, err := downloadPVs(objectStore, v.s3KeyPrefix())
 	if err != nil {
-		v.log.Error(err, fmt.Sprintf("error fetching PV cluster data from S3 profile %s", s3ProfileName))
-
 		return 0, err
```

Comment:
Why not add context to the error?

    return 0, fmt.Errorf("error restoring PVs from profile %q: %w", s3ProfileName, err)

Reply:
@nirs I am adding context in the caller function. It is now the same in both restorePVsFromObjectStore and restorePVCsFromObjectStore - these functions are returning core error.